ex2.js

```diff
@@ -0,0 +1,17 @@
+/*Scrieți o funcție pentru a genera un număr aleatoriu mai mare sau egal 
+cu o variabilă start și strict mai mic decât o variabilă end . 
+Aceste două variabile sunt capetele intervalului pentru numărul general: [start, end)
+*/
+
+const getRandomNumber = (start, end) => {
+    return Math.random () * (end - start ) + start;
+}
+console.log(getRandomNumber(3, 5)); // Nu ne va da un numar intreg 
+
+const getRandomNumber2 = (start, end) => {
+    return Math.floor(Math.random () * (end - start )) + start;
```

Comment:
```suggestion
    return Math.floor(getRandomNumber(start, end));
```
Aici ai cod care se repeata.
Se recomanda sa folosesti functia de mai sus, care face acelasi lucru, mai putin partea cu floor